UPGRADE-3.x.md

```diff
@@ -1,6 +1,40 @@
 UPGRADE 3.x
 ===========
 
+UPGRADE FROM 3.x to 3.x
+=======================
+
+### Sonata\DoctrineORMAdminBundle\Filter\CallbackFilter
+
+Deprecate the usage of a callback which does not return a boolean value.
+
+Previously, this was valid:
+```php
+function callback(ProxyQueryInterface $queryBuilder, string $alias, string $field, array $value)
```

Comment:
```suggestion
static function callback(ProxyQueryInterface $queryBuilder, string $alias, string $field, array $value)
```

Reply:
This code is out of a class in my example, I prefer to avoid using static then.